### PR TITLE
Fix the patch disc loss

### PR DESCRIPTION
**Pull request by gabrieltseng**
I am very very very very sorry 

https://wandb.ai/eai-ai2/helios-debug/runs/9jvh82hy

### CODE REVIEW

File: helios/nn/flexihelios.py [hunk 1]
```diff
@@ -1303,7 +1303,7 @@ class PredictorConfig(Config):
     num_heads: int = 2
     max_sequence_length: int = 12
     drop_path: float = 0.0
-    learnable_channel_embeddings: bool = False
+    learnable_channel_embeddings: bool = True
```
Comment (gabrieltseng) [inline helios/nn/flexihelios.py:1306]: I think this is important  - otherwise the predictor has no way of knowing which mask tokens are relevant for a particular channel group

File: helios/train/loss.py [hunk 2]
```diff
@@ -138,7 +138,8 @@ def compute(
 
         # emulate averaging across the batch dimension
         loss_multiplier = self._expand_and_reciprocate(count)
-        loss = (loss * loss_multiplier).sum() / bs
+        # can't use bs here since this is after the unsqueezing, so bs == 1
+        loss = (loss * loss_multiplier).sum() / all_preds.shape[0]
```
Comment (gabrieltseng) [inline helios/train/loss.py:142]: :facepalm: :facepalm: :facepalm:

File: helios/train/train_module/latent_mim.py [hunk 3]
```diff
@@ -46,9 +46,8 @@ class LatentMIMTrainModuleConfig(HeliosTrainModuleConfig):
     masking_config: MaskingConfig = field(
         default_factory=lambda: MaskingConfig(strategy_config={"type": "random"})
     )
-    ema_decay: float = 0.99
-    # TODO: currently causes runs to fail
-    # max_grad_norm: float = 1.0
+    ema_decay: float = 0.999
```
Comment (gabrieltseng) [inline helios/train/train_module/latent_mim.py:49]: In the original codebase (and in LatentMIM and IJEPA) the ema decay linearly increases from 0.996 to 1.0 by the end of training. This is probably what we want too.

File: helios/train/loss.py (hunk 2 above)
Comment (Hgherzog) [reply helios/train/loss.py:142]: Can we add a test here that ensures this loss does not scale with batch size

File: helios/train/train_module/latent_mim.py (hunk 3 above)
Comment (Hgherzog) [reply helios/train/train_module/latent_mim.py:49]: I will create an issue for this

File: helios/train/loss.py (hunk 2 above)
Comment (gabrieltseng) [reply helios/train/loss.py:142]: added
